headers/functions/arm9.h

```diff
@@ -968,6 +968,13 @@ void EnableVramBanksInSet(struct vram_banks_set* vram_banks);
 void GeomMtxLoad4x3(struct matrix_4x3* matrix);
 void GeomMtxMult4x3(struct matrix_4x3* matrix);
 void GeomGxFifoSendMtx4x3(struct matrix_4x3* matrix, void* gxfifo);
+int IncrementThreadCount(void);
+void InsertThreadIntoList(struct thread* thread);
+void ThreadStart(struct thread* thread, thread_entry_fn_t function, struct thread* param_3,
```

Comment:
I'm gonna change this to `StartThread`, since it does stuff _to/with_ a thread (like `InitThread`) rather than being done _by_ a thread (like `ThreadExit`). Lmk if you object.